### DISCUSSION TITLE
Application got declined 19 times

### DISCUSSION THREAD

**Discussion by theIuser** [community/community#51669] [GitHub Education]
They declined my application 19 times.

What I tried so far:
- Uploading my student ID card front and back. Declined because of picture manipulation.
- Added my real name and a bio to my profile
- Took a picture of my class schedule.
- Took a picture of my schools student portal
- Took a picture with my webcam of my student ID
- Applied from my schools wifi with browser location enabled

Here is a list of all the errors I got so far:
- Unfortunately, we weren't able to approve your educational discount request
What happened? Common verification problems and suggested solutions are documented here: 

- The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.

- The image you used appears to be edited. Manipulated images may not be used. Please try a different image.
Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

- You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.
Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?

What am I doing wrong?

**Comment by Ntshangase** [comment]
Hi @theIuser 

I would recommend that you try  setting your account activity to public and make sure that the picture quality is good.

**Reply by theIuser** [reply]
Sadly this didn't work as well. Set my profile to public. Logged out and in. Application got denied in a matter of seconds without any useful error. :(

**Comment by Fin1704** [comment]
Have you registered successfully? If so, it may not continue to be approved

**Reply by theIuser** [reply]
Hi,
What do you mean by "registered successfully"?
I used the Github account I created years ago and applied for the Education program and got declined multiple times.

Do I need to create a new account before I apply?

**Comment by samus-aran** [comment]
Hi @theIuser 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

Let's break down these rejection reasons:

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps.

2. If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

3. The application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Hope this helps! Let us know how you get along 👍

**Reply by theIuser** [reply]
@samus-aran thanks for your reply.

I totally understand that checking multiple piece of documents provided by different schools around the globe is not an easy thing.
That's why I tried to upload the same documents as my fellow students, which got accepted, did.

What concerns me the most, all of my later applications, nr 2-19 got declined with in minutes after the application. Which states that its an all automated process. Maybe if just a human could look at my application it could be approved in a short time.

All the decline messages a really vague and none specific. I'm happy to provide you with addition evidence if needed.

**Reply by theIuser** [reply]
@samus-aran Is there a way to contact support so a human can verify my application? I really tried everything possible to get my account approved.

**Reply by samus-aran** [reply]
Hi @theIuser 👋 

All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants. We process _millions_ of students every year so we can not review individuals' pieces of evidence.

What I would suggest is that you provide an alternative piece of evidence rather than you Student ID and let us know how you get on doing so. Do you have another form of evidence to verify your student status? Do try and acquire evidence that is positioned higher rather than lower in that option list.

**Comment by vixonblues23** [comment]
I have been having the issue and ready to give up on this altogether.

**Comment by vixonblues23** [comment]
There is one more thing I got to add, this is the worst program I have ever tried to get approved for. I have never had such a hard time for getting approved for something school related. I cannot I have to install this program for my informatics class. Honestly I wish this program never existed.

**Reply by samus-aran** [reply]
Hi @vixonblues23 

As I have stated earlier the verification process needs to reviews hundreds, thousands of different pieces of evidences from many institutions. However sometimes we may require a little more evidence from the student applying and we thank you for being patient. We also need to highlight that GitHub Education Student Developer Pack is not a requirement to receiving a certification at any institution and that perhaps needs to be acknowledged by your lecturers. 

Why don't you open up a separate discussion and we can converse there and see how we can assist ⭐

**Reply by vixonblues23** [reply]
That would be great, this has frustrated me to no end trying to get accepted. I messaged my teacher how much trouble I am having to get this setup. I just want to make sure I am sending the right document to get approved.

**Reply by vixonblues23** [reply]
So, how do I create a new discussion with this stupid thing? Why do we have to go through this crap to get approved? Can't they just approve you instead of trying to load documents over and over, and still get denied. This is the stupidest program I have ever heard of!

**Reply by emillienemann** [reply]
> Hi @vixonblues23
> 
> As I have stated earlier the verification process needs to reviews hundreds, thousands of different pieces of evidences from many institutions. However sometimes we may require a little more evidence from the student applying and we thank you for being patient. We also need to highlight that GitHub Education Student Developer Pack is not a requirement to receiving a certification at any institution and that perhaps needs to be acknowledged by your lecturers.
> 
> Why don't you open up a separate discussion and we can converse there and see how we can assist star

I totally get that, but don't you think it would be reasonable to at least manually look through applications for people that have been declined 10+ times? At that point, it's clear your system is wrong.
